enclave/core/sgx/globals.c

```diff
@@ -184,14 +184,23 @@ size_t __oe_get_reloc_size()
 /*
 **==============================================================================
 **
-** Extended enclave initialization data boundaries:
+** Extended enclave initialization data:
 **
 **==============================================================================
 */
 
-const void* __oe_get_eeid()
+bool oe_have_eeid(void)
 {
-    return oe_eeid;
+    return _eeid_rva != 0;
```

Comment:
`_eeid_rva` is measured and indicats whether the base enclave is signed as an EEID enclave. Should add a comment to explain the design.

Reply:
Outdated.